code/modules/assembly/igniter.dm

```diff
@@ -24,6 +24,9 @@
 	var/turf/location = get_turf(loc)
 	if(location)
 		location.hotspot_expose(1000,1000)
+		for(var/T in location.contents)
+			var/atom/A = T
+			A.fire_act()

```

Comment:
Eh I reacted that way at first too Phil, but honestly its fine. It's just one tile and once people learn that igniters are actually dangerous now, it won't be hard to manage.